greedy_methods/fractional_knapsack.py

```diff
@@ -32,17 +45,18 @@ def frac_knapsack(vl, wt, w, n):
     TypeError: unsupported operand type(s) for /: 'str' and 'int'
     """
 
+    # sort in descending order of value/weight ratio
     r = sorted(zip(vl, wt), key=lambda x: x[0] / x[1], reverse=True)
```

Comment:
Please add this and fix the tests to match.
```suggestion
    r = sorted(zip(vl, wt), key=lambda x: x[0] / x[1], reverse=True, strict=True)
```